test/timeseries_path_partitioner/processor-spec.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const { DataEntity } = require('@terascope/utils');
+const { WorkerTestHarness } = require('teraslice-test-harness');
+
+describe('Timeseries path partitioner', () => {
+    let harness;
+
+    const data = [
+        DataEntity.make(
+            {
+                date: '2020-01-17T19:21:52.159Z',
+                text: 'test data'
+            }
+        )
+    ];
+
+    beforeAll(async () => {
+        harness = WorkerTestHarness.testProcessor({
+            _op: 'timeseries_path_partitioner',
+            base_path: '/data',
+            date_field: 'date',
+            prefix: '',
+            type: 'daily'
+        }, {});
+        await harness.initialize();
+    });
+
+    afterAll(async () => {
```

Comment:
This afterAll should be a afterEach since it should cleanup after each new instance of the harness (which are created in your it blocks)

Reply:
Fixed